### ISSUE TITLE
lt1: genau wie bei der logistic, die softmax funktion "extra" als math. prinzip einführen

### ISSUE THREAD

**Issue by berndbischl** [slds-lmu/lecture_i2ml#1017]
https://en.wikipedia.org/wiki/Softmax_function

wir brauchen die definition, eigenschaften. und sollten dann zeigen wie das mit log regr, lda zusammenhängt.

und was der verlust ist --> cross entropy

**Comment by lisa-wm** [comment]
merged comments into issue 997 on same topic
